File: prob_1149/prob_1149.c
#include <stdio.h>

int main()
{
	int a, n, i, aux, soma=0;
	scanf("%d%d",&a,&n);
	
	if(n <= 0)
	{
		while(n <= 0)
			scanf("%d", &n);
	}
	
	
	for(i = 0; i < n; i++)
	{
		aux = i + a;
		soma += aux;
	}
	
	printf("%d\n", soma);
	
	return 0;
}

File: prob_1149/prob_1149.py
x = input().split(" ")

a = int(x[0])
n = int(x[1])

soma = 0
aux = 2

if(n <= 0):
    while(n <= 0):
        n = int(x[aux])
        aux += 1

for i in range(n):
    aux = i + a
    soma += aux

print("%d" %soma)

